src/java/org/apache/nutch/api/impl/RAMConfManager.java

```diff
@@ -42,6 +42,10 @@ public RAMConfManager() {
     configurations.put(ConfigResource.DEFAULT, NutchConfiguration.create());
   }
 
+  public RAMConfManager(String confId, Configuration configuration) {
```

Comment:
Javadoc on all public methods please. Can you update all of the code you've proposed so far with Javadoc.